pom.xml

```diff
@@ -928,6 +928,18 @@
         <version>0.11</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>com.github.stefanbirkner</groupId>
```

Comment:
Ur, @marblejenka . If possible, could you avoid to introduce new test dependency? Is it inevitable?

Reply:
It is not impossible, but we have to do the same thing to test code that uses environment variables. Consequently, we will include a copy of that dependency (specifically https://github.com/stefanbirkner/system-rules/blob/master/src/main/java/org/junit/contrib/java/lang/system/EnvironmentVariables.java , which enables to write environment variable via reflection api) in our test code. 
The other way is giving up coverage, but I think it not desirable.

In my opinion, system-rules is beneficial for testing of environment variables in other usecase. Additionally, there should be no problem with the license, so I would like you to merge this.

Reply:
`environmentVariables.set` is used only twice. I guessed the following will be enough for this PR. What do you think?
```java
    try {
      System.setProperty("SPARK_JARS_DIR", "...");
      // test code
    } finally {
      System.clearProperty("SPARK_JARS_DIR");
    }
```

Reply:
System#getProperties and System#getenv are slightly different. Since I use the environment variable, we can't use the suggested way(I choose System#getenv because SPARK_CONF_DIR also uses environment variable). 
https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getProperties--
https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getenv--

If adding new dependencies in this PR is unacceptable, I will open another PR aimed at improving test coverage using environment variables.
It seems that some code that uses environment variables is not branch-covered according to a coverage report I created. While improving branch coverage of existing stable code may not be very useful, I don't think it is harmful.